crypto/nrf_cc310_platform/src/nrf_cc3xx_platform_mutex_zephyr.c

```diff
@@ -172,6 +172,20 @@ nrf_cc3xx_platform_mutex_t heap_mutex = {
                     NRF_CC3XX_PLATFORM_MUTEX_MASK_IS_VALID
 };
 
+static bool mutex_flags_unknown(uint32_t flags){
+    switch(flags){
+        case (NRF_CC3XX_PLATFORM_MUTEX_MASK_IS_VALID | NRF_CC3XX_PLATFORM_MUTEX_MASK_IS_INTERNAL_MUTEX):
+        case (NRF_CC3XX_PLATFORM_MUTEX_MASK_IS_VALID | NRF_CC3XX_PLATFORM_MUTEX_MASK_IS_ALLOCATED):
+        case (NRF_CC3XX_PLATFORM_MUTEX_MASK_IS_VALID):
+        case NRF_CC3XX_PLATFORM_MUTEX_MASK_INVALID:
+        case NRF_CC3XX_PLATFORM_MUTEX_MASK_IS_HW_MUTEX:
```

Comment:
@Vge0rge to comment on this

Reply:
Could you elaborate why this is impossible? If you set the configuration so that this is enabled:
https://github.com/nrfconnect/sdk-nrfxlib/blob/424e68ba53601b087fb70d5efae32b9a6db97595/crypto/nrf_cc310_platform/src/nrf_cc3xx_platform_mutex_zephyr.c#L155

This is a possible scenario I believe.

Reply:
Code is overly complicated due to binary distribution. Not fixable.